Mediapipe.Net/Util/ResourceManager.cs

```diff
@@ -19,7 +18,8 @@ public unsafe abstract class ResourceManager
     {
         public delegate string PathResolver(string path);
         public abstract PathResolver ResolvePath { get; }
-        public delegate bool ResourceProvider(string path, void* output);
+
+        public delegate byte[] ResourceProvider(string path);
```

Comment:
I think we should still do something like
```cs
public delegate bool ResourceProvider(string path, out byte[] bytes);
```

since internally it always returns true and we want to let mediapipe know when we failed to locate a resource

Reply:
What if we throw an exception instead? I think it'll be safer.